app/custom-oauth/server/custom_oauth_server.js

```diff
@@ -254,9 +254,18 @@ export class CustomOAuth {
 
 			const serviceData = {
 				_OAuthCustom: true,
-				accessToken,
+				accessToken: response.access_token,
+				idToken: response.id_token,
+				expiresAt: (+new Date) + (1000 * parseInt(response.expires_in, 10)),
```

Comment:
We may need to add something to actually honor this.  I'm looking through meteors code base and don't see anything that even honors expiresAt :(

Reply:
I'll have a look tomorrow, and create a pull request to use expiresAt and refreshToken, if I can't find it's already there.

Btw. can you point me to some docu on how to build a Docker container from my changed sources / git.

Reply:
do `meteor build --directory /folder/you/want/to/build/in`

Then `cp .docker/Dockerfile /folder/you/want/to/build/in`

finally: go into the folder and `docker build -t theimage:tag .`

Should do the trick